lib/media_assets_manager/media_assets_manager.ex

```diff
@@ -0,0 +1,32 @@
+defmodule QbBackend.MediaAssetsManager do
+  @moduledoc """
+  This is a boundary module used to deal with handling media assets for the Quick-bytes Application.
+  Use this module to handle any media assets used in the application such as images and videos.
+  """
+  alias ExAws.S3
+   def upload_asset(asset_binary) do
+    asset_bucket = "qb-media-bucket"
+     {:ok, binary_data} = Base.decode64(asset_binary)
+     filename =
+      binary_data
+      |> image_extension()
+      |> unique_filename()
+
+
+    case do_upload(asset_bucket, filename, binary_data) do
+      {:ok, _} ->
+        {:ok, "https://#{asset_bucket}.s3.amazonaws.com/#{asset_bucket}/#{filename}"}
```

Comment:
@br4inii would you mind running mix format on this file to make it easier to read, also please consider constructing the `URL` before putting it in the result. Have you tried this on iex? does it give you back a url?